README.md

```diff
@@ -78,16 +78,17 @@ use the [cleanlab](https://github.com/cleanlab/cleanlab/) package.
+| 2 | Near Duplicates  | Images that are visually almost identical                       | near_duplicates  | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/near_duplicates.png)                                                                                                                                                  |
+| 3 | Blurry           | Images where details are fuzzy (out of focus)                   | blurry           | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/blurry.png)                                                                                                                                                           |
+| 4 | Low Information  | Images lacking content (little entropy in pixel values)         | low_information  | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/low_information.png)                                                                                                                                                  |
+| 5 | Dark             | Irregularly dark images (*under*exposed)                        | dark             | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/dark.jpg)                                                                                                                                                             |
+| 6 | Light            | Irregularly bright images (*over*exposed)                       | light            | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/light.jpg)                                                                                                                                                            |
+| 7 | Grayscale        | Images lacking color                                            | grayscale        | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/grayscale.jpg)                                                                                                                                                        |
+| 8 | Odd Aspect Ratio | Images with an unusual aspect ratio (overly skinny/wide)        | odd_aspect_ratio | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/odd_aspect_ratio.jpg)                                                                                                                                                 |
+| 9 | Size      | Images which are n times larger or smaller than the median size | size             | ![](https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/odd_aspect_ratio.jpg) <img src="https://raw.githubusercontent.com/cleanlab/assets/master/cleanvision/example_issue_images/odd_aspect_ratio.jpg" width=20% height=20%> |
```

Comment:
Change name of issue type here to `Odd Size`, also suggest a good example image to use for this issue type

Reply:
Changed the name. What do you think would be a good example image?

Reply:
I would suggest a very small image, since we can't show a very large image due to space constraints. You could resize one of the images in the test dataset to a very small size and add it here.

Reply:
Hi, I simply used one of the used images and rescaled it in the markdown.

Looks like this now:
![image](https://github.com/cleanlab/cleanvision/assets/2640499/9cdc1a55-ae9f-4581-b63f-1a954a9236e0)

I think the example images are hosted in another repo if I am not wrong.

Reply:
Could you use a different image since this one is already used in the examples? Please raise a PR in this repo https://github.com/cleanlab/assets/tree/master/cleanvision/example_issue_images for example image

Reply:
Done:
https://github.com/cleanlab/assets/pull/3

Reply:
Merged

Reply:
![image](https://github.com/cleanlab/cleanvision/assets/2640499/27411409-68be-46cc-ae9a-82036df1f479)